client/my-sites/draft/index.jsx

```diff
@@ -196,6 +196,11 @@ module.exports = React.createClass( {
 						{ post.format === 'aside' ? excerpt : title }
 					</a>
 				</h3>
+				{ post.excerpt &&
```

Comment:
Should be displaying like this:

![image](https://cloud.githubusercontent.com/assets/548849/23375337/4cb04b9a-fd28-11e6-9e28-6f4589d6a012.png)